tsl/test/expected/hypercore.out

```diff
@@ -138,17 +138,32 @@ WHERE time = '2022-06-01'::timestamptz;
 
 -- Create a new index on a compressed column
 CREATE INDEX ON readings (location);
--- Check that we error out on unsupported index types
+-- Run these checks as superuser to avoid permission errors. These
+-- commands should still be blocked and have reasonable error
+-- messages.
+set role :ROLE_SUPERUSER;
 \set ON_ERROR_STOP 0
+-- Check that we error out on unsupported index types
 create index on readings using brin (device);
 ERROR:  index access method "brin" not supported
 create index on readings using gin (jdata);
 ERROR:  index access method "gin" not supported
 create index on readings using magicam (device);
 ERROR:  access method "magicam" does not exist
+-- Check that we error out when trying to build index concurrently.
+create index concurrently on readings (device);
+ERROR:  hypertables do not support concurrent index creation
+create index concurrently invalid_index on :chunk (device);
+ERROR:  concurrent index builds not supported
+-- This will also create the index on the chunk (this is how it works,
+-- see validate_index() in index.c for more information), but that
+-- index is not valid, so we just drop it explicitly here to keep the
+-- rest of the test clean.
+drop index _timescaledb_internal.invalid_index;
```

Comment:
Hmm, should the invalid index really be there after an error in the first TX?

I understand that there might be an invalid index if something fails during a concurrent index build in tx1 or tx2.

Reply:
> Hmm, should the invalid index really be there after an error in the first TX?

"Should" is a relative term. This is how PostgreSQL behaves. If there is an error in the validation phase, the index is still in the catalog. See [`validate_index()` in `index.c`](https://github.com/postgres/postgres/blob/ea792bfd93ab8ad4ef4e3d1a741b8595db143677/src/backend/catalog/index.c#L3255).

> I understand that there might be an invalid index if something fails during a concurrent index build in tx1 or tx2.